layers/meta-resin-jetson/recipes-kernel/linux/linux-tegra_%.bbappend

```
@@ -4,6 +4,7 @@ FILESEXTRAPATHS_append := ":${THISDIR}/${PN}"
 SRC_URI_append = " \
     file://0001-Expose-spidev-to-the-userspace.patch \
     file://tegra186-tx2-cti-ASG001-USB3.dtb \
+    file://tegra186-quill-p3310-1000-c03-00-base.dtb \
```

If I'm doing the subdir change as mentioned, do I also need to change here?
```suggestion
    file://n510/tegra186-quill-p3310-1000-c03-00-base.dtb \
```